config.toml

```
@@ -0,0 +1,6 @@
+baseURL = "https://unbalancedparentheses.github.io/compiler-team/"
+languageCode = "en-us"
+title = "Rust Lang - Compiler Team"
```

We can avoid the extra `--theme` parameter if we specify it here.
```suggestion
title = "Rust Lang - Compiler Team"
theme = "book"
```